pages/index.js

```diff
@@ -12,22 +12,22 @@ export default function Home({ people, preview }) {
         <title>Homepage</title>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
       </Head>
-      <main>
+      <main className='px-4 py-4'>
```

Comment:
I think everything looks good on this file!

Reply:
I was thinking of DRYing it up a bit and making like a little `Subhead` component that would own the tailwind styles.  Like:

```js
function Subhead({ children }) {
  return (
    <h2 className='text-xl my-4 font-semibold'>
      { children }
    </h2>
  )
}
```

So that I could just do:

```js
  <Subhead>Styling using Linaria</Subhead>
```

Cause I didn't like redefining the h2 over and over again.  Is that what you'd do?  Or would you use one of the other strategies from https://tailwindcss.com/docs/reusing-styles or something else entirely?